spec/formatting.md

```diff
@@ -942,9 +942,9 @@ The _Default Bidi Strategy_ is defined as follows:
       True if the `u:dir` _option_ of the _resolved value_ of `exp` has a value other than `'inherit'`,
       or False otherwise.
    1. If `dir` is `'LTR'`:
-      1. If `msgdir` is `'LTR'` in the formatted output
+      1. If `msgdir` is `'LTR'`
          and `isolate` is False,
-         let `fmt` be itself
+         append `fmt` to the formatted output.
```

Comment:
The other steps don't do append. They talk about what `fmt` contains (e.g. "prefix `fmt` with...")

The change we should be making should actually be earlier. Something like:

> 2. For each expression `exp` in _pattern_:
>    i. Let `fmt` be the formatted string representation of the _resolved value_ of `exp` to be appended to the formatted output.